app/services/IdentityService.scala

```diff
@@ -26,7 +26,15 @@ class IdentityService(identityApiClient: => IdentityApiClient) extends LazyLoggi
 
   def doesUserExist(email: String): Future[Boolean] =
     identityApiClient.userLookupByEmail(email).map { response =>
-      (response.json \ "user" \ "id").asOpt[String].isDefined
+      response.status match {
+        case status if status == Status.OK =>  true
```

Comment:
I may be wrong, but I think you can just say `case Status.OK =>` and it'll just check equality.  You only need to name it if you're using it somewhere.